docs/deploying-airbyte/local-deployment.md

```diff
@@ -18,6 +18,16 @@ docker-compose up
 * In your browser, just visit [http://localhost:8000](http://localhost:8000)
 * Start moving some data!
 
+## Deploy on Windows
+
+We recommend follow [this](https://docs.docker.com/docker-for-windows/install/) steps to install Docker on Windows.
+After installing WSL 2 backend and `docker` you should be able to run containers using Windows PowerShell. 
+The steps on documentation suggested already install `docker-compose`. 
+If you don't have `git` installed in your machine you can download the latest Airbyte release [here](https://github.com/airbytehq/airbyte/releases).
```

Comment:
```suggestion
Instead of cloning the repo, you can alternatively download the latest Airbyte release [here](https://github.com/airbytehq/airbyte/releases).
```